metals/src/main/scala/scala/meta/internal/metals/doctor/DoctorResults.scala

```diff
@@ -29,6 +29,7 @@ final case class DoctorFolderResults(
     messages: Option[List[DoctorMessage]],
     targets: Option[Seq[DoctorTargetInfo]],
     explanations: List[Obj],
+    errorReports: List[ErrorReportInfo],
 ) {
   def toJson: Obj = {
     val json = ujson.Obj(
```

Comment:
Can you also make sure to add these to the `json` output? If not clients that are using the json Doctor won't get any of this.

Reply:
Yes! Thanks for that.